engine/apps/api/permissions.py

```diff
@@ -301,40 +309,6 @@ def has_object_permission(self, request: AuthenticatedRequest, view: ViewSetOrAP
         return True
 
 
-class BasicRolePermission(permissions.BasePermission):
```

Comment:
is it safe to delete this? can it break existing OSS installations that don't have RBAC?

Reply:
the [`user_is_authorized`](https://github.com/grafana/oncall/pull/5099/files#diff-83058af8678d629be3d878a5947537c15c01abf6ef2eb7ba31d08b8febda0853R137-R141) function already handles this scenario (which is used by the `RBACPermission` class) 🙂